Taskfile.yml

```diff
@@ -206,17 +208,22 @@ tasks:
       vars:
         COMPONENT: "{{.TASK}}"
 
-  log-viewer-webui-client:
+  log-viewer-webui-clients:
     vars:
       CHECKSUM_FILE: "{{.G_BUILD_DIR}}/{{.TASK}}.md5"
       OUTPUT_DIR: "{{.G_LOG_VIEWER_WEBUI_BUILD_DIR}}"
     sources:
-      - "{{.G_BUILD_DIR}}/log-viewer-modules.md5"
+      - "{{.G_BUILD_DIR}}/log-viewer-webui-node-modules.md5"
       - "{{.TASKFILE}}"
       - "client/package.json"
       - "client/src/**/*.css"
       - "client/src/**/*.jsx"
       - "client/src/webpack.config.js"
+      - "yscope-log-viewer/customized-packages/**/*"
+      - "yscope-log-viewer/package.json"
```

Comment:
* I think both the client and yscope-log-viewer's package.json are tracked by the node-modules task, so can we drop them from this task?
* Should we track `yscope-log-viewer/.babelrc`?

Reply:
* There could be configuration changes (e.g. babel, browserlist) in `package.json` that would result in different generated (build) assets, whereas the node_modules might remain the same and doesn't necessarily changes the log-viewer-webui-node-modules.md5 checksum.
* For the same reason, we should.

Reply:
> doesn't necessarily change the log-viewer-webui-node-modules.md5 checksum.

True, true.